### PR TITLE
Typo fix & env bump

### PR DESCRIPTION
**Pull request by Raecaug**
Fixed typo of variable name in documentation.
Added Replicate and Google Cloud env variables for new multi-modal functions.

### CODE REVIEW

File: detailed_guides/MULTI-MODALITY.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ For this functionality to work, you need a replicate account, and a correspondin
 
 The cost to run replicate for image understanding is roughly $0.0032 per second, it will take on average 0.5-1.0 seconds per image. This is a small cost, but it will add up over time, so it's not recommended to release this feature to the public unless you're comfortable with it or have billing limits set on your replicate account.
 
-As a second part of multi-modality, the bot will do OCR on uploaded images. You need to have the 'Google Cloud Vision' API enabled, and put your API key in the `GOOGLE_SEARCH_API_KEY` field in your `.env` file for this to work.
\ No newline at end of file
+As a second part of multi-modality, the bot will do OCR on uploaded images. You need to have the 'Google Cloud Vision' API enabled, and put your API key in the `GOOGLE_CLOUD_PROJECT_ID` field in your `.env` file for this to work.
```
Comment (Kav-K) [inline detailed_guides/MULTI-MODALITY.md:9]: Actually we don't need the project ID

**Review by Kav-K — CHANGES_REQUESTED** [review]

File: detailed_guides/MULTI-MODALITY.md (hunk 1 above)
Comment (Kav-K) [reply detailed_guides/MULTI-MODALITY.md:9]: oh I made this function but it isn't actually used anywhere if you check

Comment (Raecaug) [reply detailed_guides/MULTI-MODALITY.md]: Ah okay, understood. Apologies then, my mistake!

**Review by Raecaug — COMMENTED** [review]
Fixed.

**Review by Kav-K — APPROVED** [review]

**Pull request merged**